<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DMIPCNOI14007" name="TestCases.Mian_Class" tests="1" failures="0" timestamp="05 Jan 2018 09:19:22 GMT" time="0.010" errors="1">
  <testcase name="init" time="0.010" classname="TestCases.Mian_Class">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	Syntax error, insert &quot;:: IdentifierOrNew&quot; to complete ReferenceExpression
	Syntax error, insert &quot;AssignmentOperator Expression&quot; to complete Assignment
	Syntax error, insert &quot;;&quot; to complete Statement
	FileInputStream cannot be resolved to a variable
	fis cannot be resolved to a variable
	fis cannot be resolved to a variable
	fis cannot be resolved to a variable
	fis cannot be resolved to a variable
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: Syntax error, insert ":: IdentifierOrNew" to complete ReferenceExpressionSyntax error, insert "AssignmentOperator Expression" to complete AssignmentSyntax error, insert ";" to complete StatementFileInputStream cannot be resolved to a variablefis cannot be resolved to a variablefis cannot be resolved to a variablefis cannot be resolved to a variablefis cannot be resolved to a variableat dd_core.testcore.init(testcore.java:42)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- init -->
</testsuite> <!-- TestCases.Mian_Class -->
